app/grandchallenge/workstation_configs/serializers.py

```diff
@@ -66,6 +66,7 @@ class WorkstationConfigSerializer(ModelSerializer):
     default_image_interpolation = CharField(
         source="get_default_image_interpolation_display", read_only=True
     )
+    ghosting_slice_depth = IntegerField()
```

Comment:
Will try that in a bit. Why are the `default_overlay_alpha` defined as a float-field though? Directly underneath...?

Reply:
As `default_overlay_alpha` is stored as a decimal field, and the client wanted it as a number rather than the default serialisation of a string (no decimals in JSON). Do not remove them.

Reply:
:+1: